### PR TITLE
feat(dlp): added a sample for dlp inspect datastore send to scc

### PR DESCRIPTION
**Pull request by AarshD-crest**
## Description

- Added a sample for [dlp datastore send to scc](https://cloud.google.com/dlp/docs/sending-results-to-scc.md#datastore-data) with a unit test case.

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [X] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [X] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [X] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [X] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by annadata — APPROVED** [review]

File: dlp/snippets/inspect/inspect_datastore_send_to_scc.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// inspectDataStoreSendToScc inspects sensitive data in a data store and sends the results to Google Cloud Security Command Center (SCC).
```
Comment (telpirion) [inline dlp/snippets/inspect/inspect_datastore_send_to_scc.go:26]: issue: This line is too long. Split into two.

Comment (telpirion) [inline dlp/snippets/inspect/inspect_datastore_send_to_scc.go:26]: question: when this says "data store," does it refer to generic datastore or a Cloud Datastore database? If the latter, be sure to use proper capitalization for the product name.

Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_datastore_send_to_scc.go]: Fixed it.

Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_datastore_send_to_scc.go]: It is a generic Datastore so updated it with capitalized product name.

**Pull request merged**